src/transformers/configuration_utils.py

```diff
@@ -384,6 +384,11 @@ def from_pretrained(cls, pretrained_model_name_or_path: Union[str, os.PathLike],
 
         """
         config_dict, kwargs = cls.get_config_dict(pretrained_model_name_or_path, **kwargs)
+        assert (
+            config_dict["model_type"] == cls.model_type
+        ), "You tried to initiate a model of type {} with a " "pretrained model of type {}".format(
+            cls.model_type, config_dict["model_type"]
+        )
```

Comment:
```suggestion
        assert (
            config_dict["model_type"] == cls.model_type
        ), f"You tried to initiate a model of type '{cls.model_type}' with a pretrained model of type '{config_dict['model_type']}'"
```
as I already tried to communicate earlier - we don't use `format`

Reply:
Hi @stas00, 
I used the commands from step 5 [here](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests). I ran `make style`, `make quality` and `make fixup` in that order. 

The suggestion you've made is using f-strings whereas I am using `format` (would have gone ahead with f-strings myself but when I looked around the codebase I saw `format` being used and assumed that  `format` is being used across the library

Reply:
If I understand it correctly, this library sets a standard forward but doesn't insist on spending short resources on making everything consistent. So if you look at most recent code it uses f-strings, hence the recommendation.

Reply:
BTW, you only need to run `make fixup` - much faster.

Reply:
Got it. Thanks for the clarification. 
I've added the change with f-strings.

Reply:
BTW, checked in with the group - it's f-strings - we just didn't have time to adjust old code - will document this somewhere.